src/libos/src/process/task/exec.rs

```diff
@@ -46,6 +46,7 @@ pub fn exec(libos_tid: pid_t, host_tid: pid_t) -> Result<i32> {
         // task may only be modified by this function; so no lock is needed
         do_exec_task(this_thread.task() as *const Task as *mut Task);
     }
+    println!("thread {:?} exit???", this_thread.tid());
```

Comment:
Can you please replace the print with log?

Reply:
Done.